src/util/general.helpers.js

```diff
@@ -141,16 +141,6 @@ export function convertToReadableDate(dateStr) {
   return `${date.getMonth() + 1}/${date.getDate()}/${date.getFullYear()}`;
 }
 
-export function getFeedbackButtonURL(facility) {
```

Comment:
instead of delete leave this commented out on bottom of file incase we need it ever

Reply:
Sure! Wasn't sure if it would be used again, but I can comment it in case the we need it in the future.

Reply:
The function is now commented out at the bottom of the file